### ISSUE TITLE
Can't divide "inherit"

### ISSUE THREAD

**Issue by cbrst** [tbck/style-kit#1]
PostCSS chokes on this:

https://github.com/tbck/style-kit/blob/master/lib/components/button/button.css#L27-L28

**Comment by tbck** [comment]
Yep... Looks like I never caught that because I'm usually defining a line height in https://github.com/tbck/style-kit/blob/master/test/theme-map.css

Probably a good idea to keep those kinds of calculations out of the default component configs and only set them in the map. Thanks crshd.
